yo/app/scripts/services/UtilsService.js

```diff
@@ -0,0 +1,33 @@
+/*jshint undef: false, unused: false, indent: 2*/
```

Comment:
Could you document this file a bit? I have no idea what it does and its name does not help at all.